tools/scripts/generate-default-site.ts

```diff
@@ -0,0 +1,76 @@
+import { useBuild } from '@pubstudio/frontend/feature-build'
+import { useSiteSource } from '@pubstudio/frontend/feature-site-store'
+import { createSite } from '@pubstudio/frontend/util-build'
+import { containerVertical, h1 } from '@pubstudio/frontend/util-builtin'
+import { DEFAULT_BREAKPOINT_ID } from '@pubstudio/frontend/util-ids'
+import { storeSite } from '@pubstudio/frontend/util-site-store'
+import { Css, CssPseudoClass, IEditorContext, ISite } from '@pubstudio/shared/type-site'
+import {
+  ISiteRestore,
+  ISiteStore,
+  SiteSaveState,
+} from '@pubstudio/shared/type-site-store'
+import { ref } from 'vue'
+
+export const makeMockStore = (): ISiteStore => {
+  const siteId = ref('')
+  const saveError = ref()
+  const saveState = ref(SiteSaveState.Saved)
+
+  const initialize = async () => {}
+  const save = async (_site: ISite) => {}
+  const saveEditor = async (_editor: IEditorContext) => {}
+  const restore = async (): Promise<ISiteRestore> => {
+    return {} as ISiteRestore
+  }
+  return { siteId, saveState, saveError, initialize, save, saveEditor, restore }
+}
+
+const { initializeSite, site } = useSiteSource()
+await initializeSite({ store: makeMockStore(), siteId: undefined })
+site.value = createSite('__namespace__')
+
+const { addBuiltinComponent, setComponentCustomStyle, editSelectedComponent } = useBuild()
+
+addBuiltinComponent(containerVertical.id)
+setComponentCustomStyle(
+  {
+    pseudoClass: CssPseudoClass.Default,
+    property: Css.Width,
+    value: h1.style.custom[DEFAULT_BREAKPOINT_ID]?.[CssPseudoClass.Default]?.[
+      Css.Width
+    ] as string,
+  },
+  {
+    pseudoClass: CssPseudoClass.Default,
+    property: Css.Width,
+    value: '100%',
+  },
+)
+addBuiltinComponent(h1.id)
+editSelectedComponent({ content: 'My Site' })
+setComponentCustomStyle(undefined, {
+  pseudoClass: CssPseudoClass.Default,
+  property: Css.AlignContent,
+  value: 'center',
+})
+setComponentCustomStyle(undefined, {
+  pseudoClass: CssPseudoClass.Default,
+  property: Css.TextAlign,
+  value: 'center',
+})
+setComponentCustomStyle(undefined, {
+  pseudoClass: CssPseudoClass.Default,
+  property: Css.Margin,
+  value: '40px 0 40px 0',
+})
+
+const printSiteField = (key: string, field: string | null | undefined) => {
+  console.log(`pub const ${key}: &str = r##"${field ?? ''}"##;`)
+}
+
+const stored = storeSite(site.value)
```

Comment:
@abemscac This was a big refactor for what should have been a simple task, but I think it needed to be done. My goal was to be able to use `useBuild` from regular node scripts, so that it's possible to create and export a site. The problem was that some Vue files were in the dependency tree, as well as the local store. I was able to shift things around to make it work, the main changes are:
- Split components in `feature-build` to `libs/frontend/ui-build`
- Move components rendered in the builder (prosemirror, link tooltip, etc) to `libs/frontend/feature-render-builder`. They can't go in `ui-build` because there would be a circular dependency.
- Move site initialization (mainly `use-local-store` and `use-api-store` to a `libs/frontend/feature-site-store-init`. Now `initializeSite` takes an `ISiteStore` param, so it's possible to pass a mock to avoid local/API stores if you're writing a script (see the beginning of this file). 

If writing these site scripts becomes more common, we can add some initialization/convenience methods to improve the experience.